test-common/uk/gov/hmrc/economiccrimelevyregistration/generators/Generators.scala

```diff
@@ -90,13 +91,13 @@ trait Generators {
   def stringsWithMaxLength(maxLength: Int): Gen[String] =
     for {
       length <- choose(1, maxLength)
-      chars  <- listOfN(length, arbitrary[Char])
+      chars  <- listOfN(length, arbitrary[Char].retryUntil(!_.isWhitespace))
```

Comment:
can probably use     
`chars <- listOfN(length, alphaNumChar)` with .mkString or `str <- alphaNumStr` to get a string

Reply:
Changed the code.